app/Http/Controllers/ProductController.php

```diff
@@ -22,15 +24,15 @@ public function index()
 
         $products =  $this->productCollection->getAllProducts();
 
-        return response()->json($products, 201);
+        return ProductResource::collection(Product::all());
```

Comment:
Product:all is not needed in this line

ProductResource::collection($this->productCollection->getAllProducts())